django/contrib/staticfiles/checks.py

```diff
@@ -12,3 +14,16 @@ def check_finders(app_configs=None, **kwargs):
         else:
             errors.extend(finder_errors)
     return errors
+
+
+def check_storages(app_configs=None, **kwargs):
+    """Check that staticfiles is defined in the storages setting."""
+    if STATICFILES_STORAGE_ALIAS not in settings.STORAGES:
+        return [
+            checks.Error(
+                f"You must define a '{STATICFILES_STORAGE_ALIAS}' storage in your "
```

Comment:
This error message should match the one in the docs (it has to be the same).

Reply:
Oh nice, I didn't realise that. Thanks!